File: Sort an array without using any sorting algorithm/solve.py
class Solution:
    def sort012(self, arr, n):
        count0 = 0
        count1 = 0
        count2 = 0
        for i in range(n):
            if arr[i] == 0:
                count0 += 1
            elif arr[i] == 1:
                count1 += 1
            elif arr[i] == 2:
                count2 += 1

        j = 0
        while count0 != 0:
            arr[j] = 0
            j += 1
            count0 -= 1
        while count1 != 0:
            arr[j] = 1
            j += 1
            count1 -= 1
        while count2 != 0:
            arr[j] = 2
            j += 1
            count2 -= 1

File: Sort an array without using any sorting algorithm/solve.cpp
class Solution
{
    public static void sort012(int a[], int n)
    {
        int count0 = 0;
        int count1 = 0;
        int count2 = 0;
        for(int i=0; i<n; i++){
            if(a[i] == 0){    
                count0++;
            }
            else if(a[i] == 1){
                count1++;
            }
            else if(a[i] == 2){
                count2++;
            }
        }
       
        int j=0;
        while(count0!=0){
            a[j] = 0;
            j++;
            count0--;
        }
        while(count1!=0){
            a[j] = 1;
            j++;
            count1--;
        }
        while(count2!=0){
            a[j] = 2;
            j++;
            count2--;
        } 
    }
}
